### DISCUSSION TITLE
http://localhost:3000/publicaciones/valtervalik/chat-gpt-en-visual-studio-code

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#44] [Page Comments]
# http://localhost:3000/publicaciones/valtervalik/chat-gpt-en-visual-studio-code

Como utilizar la IA en la escritura, documentación y análisis de código en la realización de proyectos dentro del excelente editor de código Visual Studio Code, muy utilizado por la comunidad de programadores en todo el mundo.

http://localhost:3000/publicaciones/valtervalik/chat-gpt-en-visual-studio-code

**Comment by manuelernestog** [comment]
Genial el artículo bro. Felicidades 🎉🎉
